src/analyses/libraryDesc.ml

```diff
@@ -38,13 +38,16 @@ type special =
   | Assert of { exp: Cil.exp; check: bool; refine: bool; }
   | Lock of { lock: Cil.exp; try_: bool; write: bool; return_on_success: bool; }
   | Unlock of Cil.exp
+  | CondWait of { cond: Cil.exp; lock: Cil.exp }
   | ThreadCreate of { thread: Cil.exp; start_routine: Cil.exp; arg: Cil.exp; }
   | ThreadJoin of { thread: Cil.exp; ret_var: Cil.exp; }
   | ThreadExit of { ret_val: Cil.exp; }
   | Math of { fun_args: math; }
   | Memset of { dest: Cil.exp; ch: Cil.exp; count: Cil.exp; }
   | Bzero of { dest: Cil.exp; count: Cil.exp; }
+  | Memcpy of { dest: Cil.exp; src: Cil.exp } (* TODO: separate for strcpy, etc *)
   | Abort
+  | Identity of Cil.exp
```

Comment:
Maybe an explanation in what sense these are understood to be `Identity`?

Reply:
Literally the identity function, there's nothing more to it. Numerous annotating functions like `__builtin_expect` fall into this form to only aid the compiler optimizations.

Reply:
Still, I think a comment to this effect may be helpful here.

Reply:
Added!